Apply new brand on the website

jimmac opened the pull request:
See https://github.com/flatpak/flatpak.github.io/issues/537 for more details.

jimmac committed:
use Iter typeface

- part of the Flatpak rebrand

See https://github.com/flatpak/flatpak.github.io/issues/537

jimmac committed:
update logos

- part of the fresh rebrand

See https://github.com/flatpak/flatpak.github.io/issues/537

jimmac committed:
replace illustration with photo

- align with the new brand

See https://github.com/flatpak/flatpak.github.io/issues/537

jimmac committed:
less bold headings

razzeee commented:
This seems a bit off on lower resolutions 
![image](https://user-images.githubusercontent.com/5943908/172057712-037755ee-ed51-4208-a977-22685e231916.png)

razzeee commented on source/stylesheets/fonts.css:
```diff
@@ -1,127 +1,21 @@
+  font-weight: 100 900;
+  font-display: swap;
+  font-style: oblique 0deg 10deg;
+  src: url("../fonts/Inter.var.woff2?v=3.19") format("woff2");
```
"https://dl.flathub.org/repo/assets/Inter-3.19/inter.css"

@barthalion hosted Inter there for us, so it might be worth using it from the same source. But it might cause CSP problems

jimmac replied:
I've opted to go variable font, for it is the future. A single request covering all weight needs. Looking into sharing the asset across both sites in future would be nice.

razzeee commented:
Other then that, clearly an improvement in my book. Especially on readability in the footer.

jimmac committed:
logo: make scalable

- remove absolute sizing to allow to scaling

Addresses https://github.com/flatpak/flatpak.github.io/pull/538#issuecomment-1146833021
